WooCommerce/Classes/ViewRelated/Products/Add Product/ManualProductTypeOptions.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+/// View to show the manual product type creation options.
+///
+struct ManualProductTypeOptions: View {
+    private let command: ProductTypeBottomSheetListSelectorCommand
```

Comment:
Nit: we should send the `BottomSheetProductType` list here since it's the data that we need. `ProductTypeBottomSheetListSelectorCommand` type is associated with the `BottomSheetListSelectorViewController`, which is irrelevant to this view.

```suggestion
    private let items: [BottomSheetProductType]
```

Reply:
It would be even better if `BottomSheetProductType` is moved to a separate file now that it's used in two places.

Reply:
Thanks, great suggestion here. I refactored both the parameter and file separation in https://github.com/woocommerce/woocommerce-ios/pull/11859/commits/bc86823cd2e479aae81f486ed0ed660acf198879